gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EncassLinker.java

```diff
@@ -27,6 +33,18 @@ public void link(Encass encass, Bundle bundle, Bundle targetBundle) {
         if (policy == null) {
             throw new LinkerException("Could not find policy for Encapsulated Assertion: " + encass.getName() + ". Policy ID: " + encass.getPolicyId());
         }
+        Element encassPortalIntegrationElement = null;
+        Element encassPortalIntegrationEnabledElement = null;
+        try {
+            encassPortalIntegrationElement = getSingleElement(policy.getPolicyDocument(), API_PORTAL_ENCASS_INTEGRATION);
+        } catch (DocumentParseException ex) {
+            // do nothing
```

Comment:
log warning

Reply:
@uppoju why do we need to warn the user, if ApiPortalEncassIntegration element is present in policy we will read it, otherwise ignore it.

Reply:
its just for logging, if any parse exception